deploy/aws/tidb-operator/manifests/local-volume-provisioner.yaml

```diff
@@ -38,6 +38,8 @@ spec:
         - key: dedicated
           operator: Exists
           effect: "NoSchedule"
+      nodeSelector:
+        localVolume: "true"
```

Comment:
`local-ssd` would be a slightly better name. GKE adds this tag automatically now: `cloud.google.com/gke-local-ssd`